src/Components/AtbashIntro/AtbashIntro.js

```diff
@@ -1,57 +1,77 @@
-                        <p className="is-size-5">
-                            Can you guess how these letters would match up in an 
-                            English Atbash cipher?
-                        </p>
-                    </div>
+class AtbashIntro extends React.Component {
+    constructor(props) {
+        super(props);
+        this.timer = 0;
+        this.timers = []
+        this.state = {
+            clicked: [true, false, false],
+            finished: [true, false, false],
+            show: [true, false, false]
+        }
+    }
+
+    componentDidUnmount() {
+        this.timer = clearTimeout()
+    }
+
+    handleClickAt = (i) => {
+        if (this.state.clicked[i-1] === true) 
+            return
+        let newClick = Array.from(this.state.clicked)
+        newClick.splice(i-1, 1, true)
+        this.setState({
+            clicked: newClick
+        })
+    }
+
+    showContent = (i) => {
+        let newshow = Array.from(this.state.show)
+        newshow.splice(i-1, 1, true)
+        this.setState({
+            show: newshow
+        })
+        this.timers.push(setTimeout(() => {this.finishAnime(i)}, 1000)) 
+    }
+
+    finishAnime = (i) => {
+        let newFinish = Array.from(this.state.finished)
+        newFinish.splice(i-1, 1, true)
+        this.setState({
+            finished: newFinish
+        })
+    }
+
+    animateOnce = (stop) => {
+        //if condition met, stop animating
+        let animeProps = stop ? {opacity: [1, 1]} : {opacity: [0, 1],
+            translateX: [-64, 0],
+            delay: (el, i) => i * 500
+        }
+        return(animeProps)
+    }
+
+    // on mouseover, switch to 
+    render() {
+        return(
+            <section className="container mb-6">
+                <Anime {...this.animateOnce(true)}>
+                    {partOne()}
+                </Anime> 
+                <div onMouseOver={() => this.handleClickAt(2)} onClick={() => this.showContent(2)}>
+                        {this.state.show[1] ? <Anime {...this.animateOnce(this.state.finished[1])}>{partTwo()}</Anime> : clickMe("CLICK ME", this.state.clicked[1])}
                 </div>
-                <div className="column is-relative">
-                    <div className="position-letter-encodings">
-                        <LetterEncoding decodedLetter="A" encodedLetter="Z" hoverReveal={false}/> 
-                        <LetterEncoding decodedLetter="B" encodedLetter="Y" hoverReveal={true} num="1"/>
-                        <LetterEncoding decodedLetter="C" encodedLetter="X" hoverReveal={true} num="2"/>
-                        <p className="is-size-6">Hover over the letters to reveal!</p>
-                    </div>
+                <div onMouseOver={() => this.handleClickAt(3)} onClick={() => this.showContent(3)}>
+                    {this.state.show[2] ? <Anime {...this.animateOnce(this.state.finished[2])}>{partThree()}</Anime> : clickMe("LOOK HERE", this.state.clicked[2])}
                 </div>

```

Comment:
It might be a good idea to style these with the [cursor property](https://www.w3schools.com/cssref/pr_class_cursor.asp) to indicate to the user visually that these text blurbs will take action on click! I recall, though, that you were interested in triggering some animations once the user scrolls to a particular part of the page. Recall our discussion about the vertical navbar and its callback functions. Have you had a chance to play around with firing an Anime [timeline animation](https://animejs.com/documentation/#timelineBasics)?

Reply:
Thanks again, was wondering how to do the clicky guy. Addressed in 52a9946

I wasn't sure if I preferred to reveal the animations via timeline or click. I did click because I feel like that would be easier to take in for kids with slower (or faster) reading speeds, so I foregoed a timed approach.